lms/djangoapps/learner_dashboard/urls.py

```diff
@@ -6,5 +6,6 @@
 from . import views
 
 urlpatterns = [
+    url(r'^programs/(?P<programId>[0-9]+)/$', views.view_program_details, name='program_details_view'),
```

Comment:
So I'm regexing for numbers a-z, A-Z and '-'? Is that your understanding too @rlucioni ?